torchci/pages/api/flaky-tests/disable.ts

```diff
@@ -144,7 +151,25 @@ export function getPlatformsAffected(workflowJobNames: string[]): string[] {
 }
 
 export function getIssueBodyForFlakyTest(test: FlakyTestData): string {
-  const examplesURL = `https://hud.pytorch.org/flakytest?name=${test.name}&suite=${test.suite}`;
+  let examplesURL = `https://hud.pytorch.org/flakytest?name=${test.name}&suite=${test.suite}`;
+  let numRedGreen = `Over the past ${NUM_HOURS} hours, it has been determined flaky in ${test.workflowIds.length} workflow(s) with ${test.numRed} failures and ${test.numGreen} successes.`;
+  let debuggingSteps = `**Debugging instructions (after clicking on the recent samples link):**
+DO NOT BE ALARMED IF THE CI IS GREEN. We now shield flaky tests from developers so CI will thus be green but it will be harder to parse the logs.
+To find relevant log snippets:
+1. Click on the workflow logs linked above
+2. Click on the Test step of the job so that it is expanded. Otherwise, the grepping will not work.
+3. Grep for \`${test.name}\`
+4. There should be several instances run (as flaky tests are rerun in CI) from which you can study the logs.
+`;
+  if (test.numRed === undefined) {
```

Comment:
If I get it correctly, `test.numRed === undefined` means the new flaky across jobs because it was green, and red, and then green.  May be put a comment here to explain that.

There is also another complication is that what if tests that are flaky across job become flaky within one job, thus the ticket body becomes stale.  But then it would be fine at the first iteration as long as the test is disabled correctly I think

Reply:
added comment
yeah, it could also result in two issues being made, but thats also a problem that currently happens anyways... i will fix later